### DISCUSSION TITLE
math mode bold function

### DISCUSSION THREAD

**Discussion by xietuhuo** [typst/typst#3878] [Q&A] [answered]
In math mode, can *** * mark  function replace math.bold function** default? it's easy to remember, I can't write some show rule complete that fuction. Help

**Comment by Enivex** [comment, marked as answer]
No, it would be ambiguous, because * has other meanings in math. If it was possible to create such a show rule it would have to be extremely cursed.

Note that you don't have to write `math.bold`. Inside an equation just `bold` is sufficient.

**Reply by xietuhuo** [reply]
Thank you. Yes, I just write bold in math mode.
